projects/igniteui-angular/src/lib/combo/combo.component.html

```diff
@@ -8,7 +8,7 @@
     <ng-container ngProjectAs="igx-hint, [igxHint]">
         <ng-content select="igx-hint, [igxHint]"></ng-content>
     </ng-container>
-    <input igxInput #comboInput name="comboInput" type="text" [value]="value" readonly
+    <input igxInput #comboInput name="comboInput" type="text" [value]="displayValue.join(', ')" readonly
```

Comment:
For future reference - we might want to expose a way for the users to define a custom separator for the values in the combo.